src/System.Memory.Polyfill/System.Memory.Polyfill.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\..\tools\common.props" />
+  <PropertyGroup>
+    <Description>Adapters for Span APIs</Description>
+    <TargetFrameworks>netstandard1.3;netcoreapp2.1</TargetFrameworks>
+    <PackageTags>Span Memory Adapter</PackageTags>
+    <!-- disable automatic compile item inclusion so that we can set the correct metadata on TextTemplate-generated sources -->
```

Comment:
You don't actually have any of these, so you can remove this and the compile entry below.

Reply:
will do.